src/pages/components/textinput.tsx

```diff
@@ -15,90 +15,105 @@ import { TextInput, suomifiDesignTokens } from 'suomifi-ui-components';
 
 TextInput.displayName = 'TextInput';
 
-const Page = (): JSX.Element => (
-  <Translation ns={['textinput']}>
-    {(t) => (
-      <Layout sideNavData={sideNavData(t)}>
-        <SEO title={t('title')} />
-        <Heading variant="h1">{t('title')}</Heading>
+const Page: React.FC = (): React.ReactElement => {
```

Comment:
Is there a reason this needs explicit typing? Something IDE doesn't show, but shows as a warning in build etc?

Reply:
I copied the approach from the Toggle page, but it seems to be working fine with just the return type 👍

Reply:
I think we should prefer inferred types when possible to avoid any redundant code as it creates more maintenance burden.

Reply:
Removed the redundant typing from both TextInput and Toggle.